utils.py

```diff
@@ -8,9 +8,304 @@
 from functools import lru_cache
 from interventions import *
 
+# * Age of 1000 means not check on age
+# * Probability of -1 means that it is handled entirely by the code
```

Comment:
are these notes related to a variable below, or forgotten in the change?

Reply:
Added docstrings to make this clear